SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -54,9 +56,18 @@
     -->
   <Target Name="SonarQubeCategoriseProject"
           BeforeTargets="CoreCompile"
-          Condition=" $(SonarQubeTempPath) != '' AND $(SonarQubeTestProject) == '' " >
+          Condition=" $(SonarQubeTempPath) != '' " >
 
+    <!-- Fakes detection -->
     <PropertyGroup>
+      <IsFakesProject Condition="$(AssemblyName.EndsWith('.fakes', System.StringComparison.OrdinalIgnoreCase))">true</IsFakesProject>
```

Comment:
Not strictly related to this CR but there are at least 2 other similar technologies to fakes (Typemock isolators and Telerik JustMock) - should we have a look at excluding those as well?